vm/systemSmartContracts/esdt.go

```diff
@@ -2050,6 +2160,9 @@ func (e *esdt) EpochConfirmed(epoch uint32, _ uint64) {
 
 	e.flagRegisterAndSetAllRoles.SetValue(epoch >= e.registerAndSetAllRolesEnableEpoch)
 	log.Debug("ESDT register and set all roles", "enabled", e.flagRegisterAndSetAllRoles.IsSet())
+
+	e.flagBurnForAll.SetValue(epoch >= e.burnForAllEnableEpoch)
+	log.Debug("ESDT register and set all roles", "enabled", e.flagBurnForAll.IsSet())
```

Comment:
wrong log
`log.Debug("ESDT burn for all", "enabled", e.flagBurnForAll.IsSet())` *

Reply:
done